mne/io/brainvision/tests/data/test_highpass_hz.vhdr

```diff
@@ -0,0 +1,103 @@
+Brain Vision Data Exchange Header File Version 1.0
+; Data created by the Vision Recorder
```

Comment:
Yeah `difflib` probably isn't worth it unless you're looking for a cool little project. I agree it's okay as is